.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
```

Comment:
I'm rather surprised you didn't use GitHub actions for this 🙃

Reply:
I wanted to, but I thought as a team we were still using circle for most of our jobs?

Reply:
Yes, we do. For consistency, this is probably better 🤷‍♂